IPs/RSKIP87.md

```diff
@@ -0,0 +1,86 @@
+#  **Whitelisting unlimited mode**  
+
+| RSKIP          | 87                             |
+| :------------- | :----------------------------- |
+| **Title**      | Whitelisting unlimited mode |
+| **Created**    | 2018                           |
+| **Author**     | JD                            |
+| **Purpose**    | Sca, Usa                      |
+| **Layer**      | Core                           |
+| **Complexity** | 2                              |
+| **Status**     | Adopted                          |
+
+# Abstract
+
+This RSKIP proposes a new mode of whitelisting which will ease the whitelisting process.
+
+
+# Motivation
+
+Currently the whitelisting process implies re-whitelisting an address after each usage, successful or not. In order to provide a more fluid service we propose to add a new whitelisting mode which enables an address to lock BTC indefinitely until set otherwise.
+
+## Specification
+
+In order to enable this new mode we have made added 2 new methods to the bridge contract and we also deprecated the existing one.
+
+### addOneOffLockWhitelistAddress (formerly known as addLockWhitelistAddress)
+
+We have renamed the existing method to avoid confusion on the type of whitelist used.
+It works exactly as the previous one.
+```
+function addOneOffLockWhitelistAddress(string address, int256 maxValue) returns (int256)
+```
+You provide a base58 BTC address and the maximum accepted value (in satoshis), and it will return a value indicating success or not.
+
+This method will check if the address is valid, then will check if it's already whitelisted, and if it isn't it will add a new entry with the max value to the oneOff storage. 
```

Comment:
"max" => "maximum"
"then will check" => ""
"and if it isn't it will add" => "and will add"
"to the oneOff storage" => "to the one-off storage otherwise"